mix.exs

```diff
@@ -46,20 +45,28 @@ defmodule ExCheck.MixProject do
 
   defp docs do
     [
+      extras: [
+        "CHANGELOG.md": [title: "Changelog"],
+        "LICENSE.md": [title: "License"],
+        "README.md": [title: "Overview"]
+      ],
       main: "readme",
-      logo: "logo.svg",
+      assets: "assets",
+      logo: "assets/logo.svg",
       source_url: @github_url,
-      extras: ["README.md"]
+      source_ref: "v#{@version}",
+      formatters: ["html"]
     ]
   end
 
   defp package do
     [
+      description: "One task to efficiently run all code analysis & testing tools in an Elixir project",
       maintainers: ["Karol Słuszniak"],
       licenses: ["MIT"],
       links: %{
-        "GitHub repository" => @github_url,
-        "Changelog" => @github_url <> "/blob/master/CHANGELOG.md"
+        "Changelog" => "https://hexdocs.pm/ex_check/changelog.html",
+        "GitHub" => @github_url
```

Comment:
It's a detail but I actually prefer "GitHub repository" - this link is presented on Hex alongside "Online documentation" that leads to HexDocs - "GitHub" alone sounds inconsistent with that.

Reply:
Done.